doc/source/api.rst

```diff
@@ -5,6 +5,21 @@
 API Reference
 *************
 
+This page gives an overview of all public pandas objects, functions and
+methods. In general, all classes and functions exposed in the top-level
+``pandas.*`` namespace are regarded as public.
+Further some of the subpackages are public as well, including `pandas.errors``,
```

Comment:
yeah, I don't think such links would currently work. But we could make such pages. That would actually maybe not bad for the subpackages, to have one page with all functions, instead of each function on a separate page. 
But anyway, that is for another PR